schemas/openshift/cluster-1.yml

```diff
@@ -32,49 +32,51 @@ properties:
     type: string
     format: uri
   auth:
-    type: object
-    properties:
-      service:
-        type: string
-        enum:
-        - github-org
-        - github-org-team
-        - oidc
-      org:
-        type: string
-      team:
-        type: string
-    oneOf:
-    - properties:
+    type: array
+    items:
+      type: object
+      properties:
         service:
           type: string
           enum:
           - github-org
-        org:
-          type: string
-      required:
-      - service
-      - org
-    - properties:
-        service:
-          type: string
-          enum:
           - github-org-team
+          - oidc
         org:
           type: string
         team:
           type: string
-      required:
-      - service
-      - org
-      - team
-    - properties:
-        service:
-          type: string
-          enum:
-          - oidc
-      required:
-      - service
+      oneOf:
+      - properties:
+          service:
+            type: string
+            enum:
+            - github-org
+          org:
+            type: string
+        required:
+        - service
+        - org
+      - properties:
+          service:
+            type: string
+            enum:
+            - github-org-team
+          org:
+            type: string
+          team:
+            type: string
+        required:
+        - service
+        - org
+        - team
+      - properties:
+          service:
+            type: string
+            enum:
+            - oidc
+        required:
+        - service
```

Comment:
does `oidc` mean auth.redhat.com? Should we name it against the provider rather than the protocol, eg `oidc-auth-redhat`?

Reply:
This is as it was before. I'll adapt the schemas for auth.redhat.com in a later PR